vault-plugins/flant_iam/model/aliases.go

```diff
@@ -15,4 +15,5 @@ type (
 	IdentitySharingUUID        = string
 	ReplicaName                = string
 	OwnerUUID                  = string
+	UnixTime                   = int64
```

Comment:
Why not just time.Time? Wouldn't it be more convenient? Even if memdb cannot handle it, I'd rather cast it where we need, but have more convenient type in processing.

Reply:
It is a format clear for every programming language.